.github/scripts/test_open_data_assets.py

```diff
@@ -138,6 +144,8 @@ def diff_row_counts(
     athena_model_row_counts: typing.List[typing.Dict],
     open_data_asset_name: str,
     open_data_asset_row_counts: typing.List[typing.Dict],
+    athena_model_year_field: str = DEFAULT_YEAR_FIELD,
```

Comment:
We don't actually override any model year fields in this PR, but as long as I'm changing this interface I figure I might as well add support in case we need it later.